kbase-extension/static/kbase/js/narrativeConfig.js

```diff
@@ -105,62 +105,59 @@ define([
      */
     function updateConfig() {
         return new Promise.try(function (resolve, reject) {
```

Comment:
this should be just 
`new Promise(function (resolve, reject) {...`
also if anything throws in here (`assertConfig()`?) it will not cause the promise to reject, but rather the constructor will fail and throw.